cvat/apps/engine/annotation.py

```diff
@@ -818,6 +818,9 @@ def merge(self, objects, start_frame, overlap):
 
         # Nothing to merge here. Just add all int_objects if any.
         if not old_objects_by_frame or not int_objects_by_frame:
+            for old_obj in self.objects:
```

Comment:
need to use old_objects_by_frame.values()

Reply:
Done.